micro-infra-hystrix/src/main/groovy/com/ofg/infrastructure/hystrix/HystrixServletConfiguration.groovy

```diff
@@ -0,0 +1,27 @@
+package com.ofg.infrastructure.hystrix
+
+import com.netflix.hystrix.contrib.metrics.eventstream.HystrixMetricsStreamServlet
+import groovy.transform.CompileStatic
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.context.embedded.ServletRegistrationBean
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Conditional
+import org.springframework.context.annotation.Configuration
+
+/**
+ * Registers a servlet that will be streaming Hystrix data
+ *
+ * @see HystrixMetricsStreamServlet
+ */
+@CompileStatic
+@Configuration
+@Conditional(IsHystrixServletEnabled.class)
+class HystrixServletConfiguration {
+
+    @Bean
+    public ServletRegistrationBean servletRegistrationBean(
+            @Value('${hystrix.stream-servlet.path:/hystrix.stream}') String hystrixMetricsStreamPath) {
```

Comment:
Done in https://github.com/4finance/micro-infra-spring/commit/7c699ba32cfa72a71295146cfecbd1c66750a53e